testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -656,33 +649,35 @@ public void testDelete()
 
         // delete using a subquery
 
-        assertUpdate("CREATE TABLE " + tableName + " AS SELECT * FROM lineitem", "SELECT count(*) FROM lineitem");
+        assertUpdate(
+                "CREATE TABLE " + tableName + " AS SELECT * FROM nation",
+                "VALUES 25");
 
         assertUpdate(
-                "DELETE FROM " + tableName + " WHERE orderkey IN (SELECT orderkey FROM orders WHERE orderstatus = 'F')",
-                "SELECT count(*) FROM lineitem WHERE orderkey IN (SELECT orderkey FROM orders WHERE orderstatus = 'F')");
+                "DELETE FROM " + tableName + " WHERE regionkey IN (SELECT regionkey FROM region WHERE name LIKE 'A%')",
+                "VALUES 15");
         assertQuery(
                 "SELECT * FROM " + tableName,
-                "SELECT * FROM lineitem WHERE orderkey IN (SELECT orderkey FROM orders WHERE orderstatus <> 'F')");
+                "SELECT * FROM nation WHERE regionkey IN (SELECT regionkey FROM region WHERE name NOT LIKE 'A%')");
 
         assertUpdate("DROP TABLE " + tableName);
 
         // delete with multiple SemiJoin
 
-        assertUpdate("CREATE TABLE " + tableName + " AS SELECT * FROM lineitem", "SELECT count(*) FROM lineitem");
+        assertUpdate(
+                "CREATE TABLE " + tableName + " AS SELECT * FROM nation",
+                "VALUES 25");
```

Comment:
just `25` (plain int)?

Reply:
not sure what's the question?

note that this is "compare with h2" api where we don't check result types

Reply:
I mean
```
assertUpdate("CREATE TABLE " + tableName + " AS SELECT * FROM nation",  25);
```

Reply:
thanks!